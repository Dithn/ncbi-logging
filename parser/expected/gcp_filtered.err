converting GCP format
total : 1
accepted : 1
rejected : 0
unrecognized : 0
