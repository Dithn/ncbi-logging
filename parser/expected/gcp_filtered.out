{"accepted":true,"accession":"SRR1269314","agent":"GCS Lifecycle Management","bucket":"sra-pub-src-3","extension":".p1_export.txt.gzb_fixed.bam","filename":"s_3_11000","host":"","ip":"","ip_region":"","ip_type":0,"method":"PUT","operation":"PUT_Object","path":"SRR1269314/s_3_11000.p1_export.txt.gzb_fixed.bam","referer":"","request_bytes":0,"request_id":"02478f5fe3deaa079d1a4f6033043a80","result_bytes":0,"source":"GS","status":200,"time":1591062496651000,"time_taken":0,"uri":"","vdb_libc":"","vdb_os":"","vdb_phid_compute_env":"","vdb_phid_guid":"","vdb_phid_session_id":"","vdb_release":"","vdb_tool":"","vers":""}
